src/solve3.F

```diff
@@ -262,7 +262,7 @@ subroutine solve3(nstep,rbufsz,num_soil_layers,                  &
       real, intent(inout), dimension(ibl:iel,jbl:jel) :: tmn,tml,t0ml,hml,h0ml,huml,hvml,tmoml
       real, intent(inout), dimension(ibp:iep,jbp:jep,kbp:kep,npt) :: pta,pt3d,ptten
       real, intent(inout), dimension(nparcelsLocal,npvals) :: pdata
-      real, intent(inout), dimension(ib:ie,jb:je,kb:ke,2) :: dpten
+      double precision, intent(inout), dimension(ib:ie,jb:je,kb:ke,6) :: dpten
```

Comment:
I really need George's confirmation, since he apparently did things differently than what I was expecting, but the velocity feedbacks from droplets (`uten`, `vten`, `wten`) can completely fit within the ranges `ib:ie`, `jb:je`, `kb:ke`. Those extra ghost points are needed for other types of tendency terms, but not the droplet ones.

It does get a little confusing now: at the cm1.F and solve3.F levels, uten, vten, and wten have `ie+1`, `je+1`, and `ke+1` bounds, respectively. But inside droplet_driver, they can all be the same, corresponding to the size of dpten. We could perhaps consider changing the variable names `uten`, `vten`, and `wten` inside droplet.F?